app/assets/stylesheets/comments.scss

```diff
@@ -59,7 +67,7 @@
     resize: none;
   }
   textarea.comment_box:focus, textarea.comment_box:valid, textarea.comment_box:active {
-    border-color: $border-dark-grey;
+    border-color: #e6e6e6;
```

Comment:
Color literals like `#e6e6e6` should only be used in variable declarations; they should be referred to via variable everywhere else.